scripts/main/mapview.js

```diff
@@ -281,6 +285,34 @@ mapview.open = function (albumID = null) {
 		updateZoom();
 	};
 
+	const setTrack = function (album) {
+		// add track
+		if (album.track_id) {
+			mapview.trackLayer = new L.GPX("uploads/tracks/" + album.track_id + ".xml", {
+				async: true,
+				marker_options: {
+					startIconUrl: null,
+					endIconUrl: null,
+					shadowUrl: null,
+				},
+			})
+				.on("error", function (e) {
+					lychee.error(lycche.locale["ERROR_GPX"] + e.err);
+				})
+				.on("loaded", function (e) {
+					if (
+						album.photos.filter((element) => {
+							return element.latitude || element.longitude;
```

Comment:
```suggestion
							return element.latitude !== null || element.longitude !== null;
```
Please keep in mind, that the geographically position 0°0'0.000'' / 0°0'0.000'' is very unlikely but valid.